api/settings/v1/recording_rules.proto

```diff
@@ -74,11 +74,34 @@ message RecordingRule {
   repeated string matchers = 4;
   repeated string group_by = 5;
   repeated types.v1.LabelPair external_labels = 6;
+  StacktraceFilter stacktrace_filter = 7;
 
   // The observed generation of this recording rule. This value should be
   // provided when making updates to this record, to avoid conflicting
   // concurrent updates.
-  int64 generation = 7;
+  int64 generation = 8;
```

Comment:
Careful in protobuf those field IDs should never change or be reused after deprecation, as that would break existing API users.